pylot/perception/depth_estimation/depth_est_operator.py

```diff
@@ -0,0 +1,146 @@
+from collections import deque
+import cv2
+from torch.autograd import Variable
+import time
+import os
+import torch
+import torch.nn as nn
+import torch.nn.parallel
+import torch.optim as optim
+import torch.utils.data
+import torch.nn.functional as F
+import numpy as np
+import torch.backends.cudnn as cudnn
+import threading
+import matplotlib.pyplot as plt
+
+import anynet.anynet
+from anynet import preprocess
+
+from erdos.op import Op
+from erdos.utils import setup_csv_logging, setup_logging, time_epoch_ms
+
+from pylot.perception.messages import DepthFrameMessage
+from pylot.utils import add_timestamp, create_depth_est_stream, is_camera_stream, rgb_to_bgr, bgra_to_bgr
+
+args = {'init_channels':1,
+        'maxdisplist':[12, 3, 3],
+        'spn_init_channels':8,
+        'nblocks':1,
+        'layers_3d':4,
+        'channels_3d':4,
+        'growth_rate':[4,1,1],
+        'with_spn':0,
+        'resume':'/home/carla/code/pylot/dependencies/anynet/results/finetune_anynet/checkpoint.tar',
```

Comment:
Can you please use flags here? Here's an example on how we do it for traffic light detection: [https://github.com/erdos-project/pylot/blob/master/pylot/perception/detection/traffic_light_det_operator.py#L32](https://github.com/erdos-project/pylot/blob/master/pylot/perception/detection/traffic_light_det_operator.py#L32).

Reply:
for pretrained as well.